new_builder_evaluation/app/domain/evaluation.py

```diff
@@ -362,8 +404,87 @@ def evaluation(self, task: str, model_s3_zip: str, model_id: int) -> dict:
 
             self.score_repository.add(new_score)
             new_scores.append(new_score)
-            url_light_model = self.builder.create_light_model(model_name, folder_name)
-            self.model_repository.update_light_model(model_id, url_light_model)
+        for no_scoring_dataset in no_scoring_datasets:
```

Comment:
Most of this for loop is copied from above. Shouldn't we just include the not scoring datasets in that same loop and simply update the database with their respective category?

Reply:
My bad, thanks